components/board/genericlinux/data.go

```diff
@@ -144,53 +144,40 @@ func getPwmChipDefs(pinDefs []PinDefinition) (map[string]pwmChipData, error) {
 
 	// Now, look for all chips whose names we found.
 	pwmChipsInfo := map[string]pwmChipData{}
+	var found bool
+	sysPrefix := "/sys/class/pwm"
```

Comment:
This is a constant, yeah? Make that clear.
```suggestion
	const sysPrefix = "/sys/class/pwm"
```